src/material/datepicker/month-view.ts

```diff
@@ -398,6 +415,15 @@ export class MatMonthView<D> implements AfterContentInit, OnChanges, OnDestroy {
     }
   }
 
+  /** Used for turning a cell number into a date. */
+  private _normalizeDayOfMonthToActiveDate(dayOfMonth: number): D {
```

Comment:
Maybe `_getDateFromDayOfMonth`?
(similar for other "normalize" functions in this PR)

Reply:
done ✔️